tests/scripts/run-xunit-perf.py

```diff
@@ -0,0 +1,464 @@
+    log('isScenarioTest: %s' % args.isScenarioTest)
+    log('isPgoOptimized: %s' % args.isPgoOptimized)
+    log('benchviewGroup: %s' % args.isScenarioTest)
+    log('hasWarmupRun: %s' % args.hasWarmupRun)
+    log('library: %s' % args.library)
+    log('collectionFlags: %s' % args.collectionFlags)
+    log('uploadToBenchview: %s' % args.uploadToBenchview)
+
+    return (coreclrPerf, args.arch, args.operatingSystem, args.configuration, args.jitName, args.optLevel, args.runType, args.outputDir, args.stabilityPrefix, args.isScenarioTest, args.benchviewPath, args.isPgoOptimized, args.benchviewGroup, args.hasWarmupRun, args.collectionFlags, args.library, args.uploadToBenchview, args.better, args.sliceNumber, args.sliceConfigFile)
+
+def log(message):
+    """ Print logging information
+    Args:
+        message (str): message to be printed
+    """
+
+    print('[%s]: %s' % (sys.argv[0], message))
+
+def print_error(out, err, message):
+    log('%s' % out.decode("utf-8"))
+    log('%s' % err.decode("utf-8"))
+    raise Exception(message)
+
+from sys import version_info
+def is_supported_version() -> bool:
+    return version_info.major > 2 and version_info.minor > 4
+
+def run_benchmark(benchname, benchdir, env, sandboxDir, benchmarkOutputDir, testFileExt, stabilityPrefix, collectionFlags, lvRunId, etwCollection, isScenarioTest, arch, extension, executable):
+    myEnv = env
+    benchnameWithExt = benchname + '.' + testFileExt
+    fullPath = os.path.join(benchdir, benchnameWithExt)
+    shutil.copy2(fullPath, sandboxDir)
+
+    files = glob.iglob(os.path.join(benchdir, "*.txt"))
+    for filename in files:
+        if os.path.isfile(filename):
+            shutil.copy2(filename, sandboxDir)
+
+    myEnv['CORE_ROOT'] = sandboxDir
+
+    benchnameLogFileName = os.path.join(benchmarkOutputDir, lvRunId + '-' + benchname + '.log')
+
+    if not os.path.isdir(benchmarkOutputDir):
+        os.makedirs(benchmarkOutputDir)
+
+    log('')
+    log('--------')
+    log('Running %s %s' % (lvRunId, benchname))
+    log('--------')
+    log('')
+
+    lvCommonArgs = [os.path.join(sandboxDir, benchnameWithExt),
+            '--perf:outputdir',
+            benchmarkOutputDir,
+            '--perf:runid',
+            lvRunId]
+
+    if isScenarioTest:
+        lvCommonArgs += ['--target-architecture',
+                arch]
+    else:
+        lvCommonArgs.insert(0, 'PerfHarness.dll')
+
+    splitPrefix = [] if stabilityPrefix is None else stabilityPrefix.split(' ')
+    runArgs = executable + splitPrefix + [os.path.join(sandboxDir, 'corerun' + extension)] + lvCommonArgs + ['--perf:collect', collectionFlags]
+    log(" ".join(runArgs))
+    sys.stdout.flush()
+
+    error = 0
+    expectedOutputFile = os.path.join(benchmarkOutputDir, lvRunId + '-' + benchname + '.xml')
+    with open(benchnameLogFileName, 'wb') as out:
+        proc = subprocess.Popen(' '.join(runArgs), shell=True, stdout=out, stderr=out, env=myEnv)
```

Comment:
The problem here was that subprocess was eating the extra quotes that are required for start prefix. I can try this, but I'm not convinced it will work.